<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1054</width>
    <height>646</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
background-color:#455364;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QSlider" name="speedSlider">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>470</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true"> QSlider::groove:horizontal {
     background: white;
     position: absolute; /* абсолютная позиция в 4px слева и справа от виджета. установка полей виджета также будет работать... */
     left: 10px; right: 10px;
 }

 QSlider::handle:horizontal {
     height: 10px;
     background: black;
     margin: 0 -10px; /* расширяется наружу от бороздки */
 }

 QSlider::add-page:horizontal {
     background: white;
 }

 QSlider::sub-page:horizontal {
     background: red;
 }</string>
    </property>
    <property name="minimum">
     <number>800</number>
    </property>
    <property name="maximum">
     <number>2300</number>
    </property>
    <property name="value">
     <number>1550</number>
    </property>
    <property name="sliderPosition">
     <number>1550</number>
    </property>
    <property name="tracking">
     <bool>true</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdSPEED">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>470</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QSlider" name="servoSlider">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>550</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true"> QSlider::groove:horizontal {
     background: white;
     position: absolute; /* абсолютная позиция в 4px слева и справа от виджета. установка полей виджета также будет работать... */
     left: 10px; right: 10px;
 }

 QSlider::handle:horizontal {
     height: 10px;
     background: black;
     margin: 0 -10px; /* расширяется наружу от бороздки */
 }

 QSlider::add-page:horizontal {
     background: white;
 }

 QSlider::sub-page:horizontal {
     background: white;
 }</string>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>180</number>
    </property>
    <property name="value">
     <number>90</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLCDNumber" name="temperaturL01">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>211</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QLCDNumber{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0.0284091 rgba(255, 0, 79, 255), stop:1 rgba(255, 124, 55, 252));
  color: rgb(255, 255, 255);
}


</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="ledC">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>CheckBox</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="temperaturL02">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>211</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QLCDNumber{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0.0284091 rgba(255, 0, 79, 255), stop:1 rgba(255, 124, 55, 252));
  color: rgb(255, 255, 255);
}


</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdSPEED_2">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>550</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="openB">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>openB</string>
    </property>
   </widget>
   <widget class="QPushButton" name="closeB">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>80</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>closeB</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comL">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>181</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="resetB">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>resetB</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>speedSlider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>lcdSPEED</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>666</x>
     <y>471</y>
    </hint>
    <hint type="destinationlabel">
     <x>940</x>
     <y>442</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>servoSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdSPEED_2</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>870</x>
     <y>556</y>
    </hint>
    <hint type="destinationlabel">
     <x>945</x>
     <y>543</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
